magento/magento2
# [Framework] New Link is not correctly shown as Current if contains default parts

### Description (*)
This PR fixes the logic of `isCurrent` method by comparing the current requested url with the link path. This way we'll always know if the link is current, even it uses some `index` default parts. 

### Fixed Issues (if relevant)
1. magento/magento2#19099: New Link is not correctly shown as Current if contains default parts

### Manual testing scenarios (*)
1. Add a new link to customer account navigation menu
2. Navigate to the new added link
3. The link should have active state in the sidebar
![7bec67d2e5](https://user-images.githubusercontent.com/15868188/48267709-b6731980-e43b-11e8-8dae-e9517c0e3dd8.jpg)



### Contribution checklist (*)
